website/content/api-docs/secret/aws.mdx

```diff
@@ -612,7 +612,8 @@ is specified as part of the URL.
 
 - `rotation_period` `(string/int: <required>)` – Specifies the amount of time
 Vault should wait before rotating the password. The minimum is 1 minute. Can be
-specified in either `24h` or `86400` format (see [duration format strings](/vault/docs/concepts/duration-format)).
+specified in either `24h` or `86400` format (see [duration format strings](/vault/docs/concepts/duration-format)). On an update,
+this will 'reset' the next rotation to occur at `now` + `rotation_period`.
```

Comment:
```suggestion
specified in either `24h` or `86400` format (see [duration format strings](/vault/docs/concepts/duration-format)). Updating the rotation period
'resets' the next rotation to occur at `now` + `rotation_period`.
```

Style correction: write in active voice

Reply:
updated to avoid using 'this' as a subject